pom.xml

```diff
@@ -122,62 +122,63 @@
+    <create.dependency.reduced.pom>false</create.dependency.reduced.pom>
+    <fastutil.version>8.5.9</fastutil.version>
+    <findbugs.skip>false</findbugs.skip>
     <grpc.version>1.54.1</grpc.version>
     <gson.version>2.8.9</gson.version>
-    <netty.version>4.1.87.Final</netty.version>
-    <rocksdb.version>7.0.3</rocksdb.version>
+    <guava.version>31.0.1-jre</guava.version>
+    <guice.version>5.1.0</guice.version>
+    <hadoop-cos.version>3.1.0-5.8.5</hadoop-cos.version>
     <hadoop.version>3.3.1</hadoop.version>
+    <hive-metastore.version>3.1.2</hive-metastore.version>
+    <jackson.version>2.13.5</jackson.version>
     <jacoco.version>0.8.5</jacoco.version>
     <java.version>1.8</java.version>
     <jaxb.version>2.3.3</jaxb.version>
     <jersey.version>2.34</jersey.version>
     <jetty.version>9.4.46.v20220331</jetty.version>
+    <jnr-fuse.version>0.5.5</jnr-fuse.version>
     <junit.version>4.13.1</junit.version>
     <jupiter.version>5.9.2</jupiter.version>
+    <kerby.version>1.0.1</kerby.version>
+    <libcephfs.version>0.0.1</libcephfs.version>
     <log4j.version>2.17.1</log4j.version>
     <maven.version>3.8.6</maven.version>
     <metrics.version>4.1.11</metrics.version>
     <mockito.version>3.4.4</mockito.version>
+    <netty.version>4.1.87.Final</netty.version>
     <orc.version>1.6.3</orc.version>
     <oshi.version>5.3.5</oshi.version>
+    <parquet.version>1.11.0</parquet.version>
     <powermock.version>2.0.7</powermock.version>
+    <project.build.outputTimestamp>${git.commit.time}</project.build.outputTimestamp>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <prometheus.version>0.8.0</prometheus.version>
-    <guava.version>31.0.1-jre</guava.version>
-    <guice.version>5.1.0</guice.version>
-    <parquet.version>1.11.0</parquet.version>
     <protobuf.version>3.19.6</protobuf.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.build.outputTimestamp>${git.commit.time}</project.build.outputTimestamp>
-    <slf4j.version>1.7.30</slf4j.version>
-    <jackson.version>2.13.5</jackson.version>
-    <hadoop-cos.version>3.1.0-5.8.5</hadoop-cos.version>
-    <cos_api.version>5.6.19</cos_api.version>
+    <ratis.version>2.5.1</ratis.version>
+    <rocksdb.version>7.0.3</rocksdb.version>
     <s3proxy.version>2.0.0</s3proxy.version>
+    <slf4j.version>1.7.30</slf4j.version>
     <stateless4j.version>2.6.0</stateless4j.version>
-    <surefire.forkCount>2</surefire.forkCount>
-    <surefire.useSystemClassLoader>true</surefire.useSystemClassLoader>
     <surefire.excludesFile>slow-tests</surefire.excludesFile>
+    <surefire.forkCount>2</surefire.forkCount>
     <surefire.rerunFailingTestsCount>1</surefire.rerunFailingTestsCount>
+    <surefire.useSystemClassLoader>true</surefire.useSystemClassLoader>
     <test.output.redirect>true</test.output.redirect>
     <update.check.auth.string>YWxsdXhpbzp0YWNoeW9u</update.check.auth.string>
```

Comment:
@Xenorith do you know what is this auth string? looks like my gitguadian warn me about it

Reply:
it's a string used to verify the requests received by the update check server. we don't want to try to handle any request from other sources. there is no problem with exposing it in the source code